.circleci/config.yml

```diff
@@ -35,6 +35,9 @@ jobs:
         steps:
         - go/save-cache:
             key: v1-go<< parameters.go_version >>
+    - run:
+        command: make test
```

Comment:
What's about `make style`? I think we should add a run step for sigv4 above, under:
```
    - when:
        condition: << parameters.run_style >>
        steps:
        - run: make style
```

Reply:
+1